packages/react-components/react-avatar/src/components/AvatarGroup/useAvatarGroupStyles.ts

```diff
@@ -18,34 +22,179 @@ const useStyles = makeStyles({
     display: 'inline-flex',
     position: 'relative',
   },
+  pie: {
+    clipPath: 'circle(50%)',
+  },
+});
+
+/**
+ * Styles for overflow button slot.
+ */
+const useOverflowButtonStyles = makeStyles({
+  base: {
+    display: 'inline-flex',
+    position: 'relative',
+    flexShrink: 0,
+    justifyContent: 'center',
+    alignItems: 'center',
+    color: tokens.colorNeutralForeground1,
+    backgroundColor: tokens.colorNeutralBackground1,
+    ...shorthands.borderWidth(`var(${avatarGroupOverflowButtonBorderVar})`),
+    ...shorthands.borderColor(tokens.colorNeutralStroke1),
+    ...shorthands.borderRadius(tokens.borderRadiusCircular),
+    ...shorthands.borderStyle('solid'),
+    ...shorthands.padding(tokens.spacingHorizontalNone),
```

Comment:
Yes, this is necessary when the size of the button is less than 28. The padding makes it so that the font or icon do not show up. By default, the button has a padding of `1 6` and that causes some issues.

Reply:
Ah got it; it's to remove the default padding of button. You can use `...shorthands.padding(0)` in that case.. no need to use the token if the visual spec doesn't mention it.